### PR TITLE
Alexk methodology page

### LINKED ISSUE #824: Make methodology a top-level page in UP web tool
**What needs to be done? Why does it need to be done?**
Currently, our methodology is linked from each individual chart. We would like to consolidate the methodology into a top-level page element of the entire dashboard. [Mocks](https://www.figma.com/file/SOdY6JujMpbawgJ4jjDmE8/Population-Projection-MVP?node-id=861%3A687)

Detailed interactions and styles can be found in this Figma file: https://www.figma.com/file/SOdY6JujMpbawgJ4jjDmE8/Population-Projection-MVP?node-id=861%3A687. This should build on what was built [here](https://app.zenhub.com/workspace/o/recidiviz/pulse-dashboard/issues/777) – we are just adding a new top-level page. 

The Methodology page should be the last one in the default dropdown order. Note: the high-level pages will differ between the North Dakota and Idaho UP leadership dashboards, but the methodology page will be constant between the two. Regardless of the state, the Methodology page should always be the last one in the dropdown.

**Additional context**
All copy can be found in the figma file linked along with text and header styles.

The table of contents links should smoothly scroll to the linked section without reloading the page. They should also update the header bar to have an anchor link so users can copy and paste links to sections.

### PR DESCRIPTION
**Pull request by AlexK-ino**
## Description of the change

> Added a new methodology page

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #824

### Development

These boxes should be checked by the submitter prior to merging:

- [X] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Comment by AlexK-ino** [comment]
@Catacola hello!
I have encountered this situation:
I need to make a smooth scroll on the anchor and the same way that the hash stays in the location, to be able to copy the path. As it turned out, these are mutually exclusive things, or I have not found an alternative yet:

1. I added smooth scrolling with JS, but then the hash does not change
2. If hash changes - page abruptly goes to the desired block and no smoothness

Thanks!
Alexander

**Comment by AlexK-ino** [comment]
Hello @Catacola!
For some reason, Travis falls down 
Locally, all the specified commands in travis.yml
have no errors. 
I tried specifying the Node version for it, but this does not solve the problem

**Comment by Catacola** [comment]
The travis failure is due to a git issue. It wasn't seeing a file that was there due to a casing issue. If you pull `feature/core-header-refresh` and rebase on it, the issues should be resolved.

File: src/core/methodology/Methodology.scss [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      }
+      &:active {
+        color: $core-link-color;
+      }
+    }
+  }
+  &__sub-block {
+    &--title {
+      padding: 4.5rem 0px 0px 0px !important;
+      font-family: "Libre Franklin", serif;
```
Comment (Catacola) [inline src/core/methodology/Methodology.scss:52]: Baskerville

File: src/assets/styles/spec/components/pageContainer.scss [hunk 2]
```diff
@@ -48,7 +48,7 @@
 
   @include between($breakpoint-md, $breakpoint-xl) {
     left: 0;
-    padding-left: $collapsed-size;
+    // padding-left: $collapsed-size;
```
Comment (Catacola) [inline src/assets/styles/spec/components/pageContainer.scss:51]: Please remove

**Review by Catacola — CHANGES_REQUESTED** [review]
This is looking very promising, thanks! Just a few minor notes besides the rebase.

**Comment by Catacola** [comment]
Looks good! I'll go ahead and add it to our overall feature branch

**Pull request merged**